string mission_name
---
string message
string Mission_Name
navigation_yeon/Mission[] Mission
